.readme-partials.yaml

```diff
@@ -0,0 +1,10 @@
+introduction: |-
+    > Node.js idiomatic client for [Cloud Storage][product-docs].
```

Comment:
should this read cloud security and not cloud storage?

Reply:
🤦‍♂